### PR TITLE
Add Caranthir Tower Reborn Quest Expansion

### PR DESCRIPTION
**Pull request by cecell**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -20951,6 +20951,15 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'Caranthir Tower Reborn Expanded.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/62755' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: The URL is missing a trailing slash:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/62755/' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -20951,6 +20951,15 @@ plugins:
...
+    dirty:
+      # version: 1
+      - <<: *quickClean
+        crc: 0x63903DC2
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 29
+  
   - name: 'Castle Volkihar Rebuilt.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]: There's some trailing whitespace:

```suggestion
        itm: 29

  - name: 'Castle Volkihar Rebuilt.esp'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -20951,6 +20951,15 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'Caranthir Tower Reborn Expanded.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/62755' ]
+    dirty:
+      # version: 1
```
Comment (sibir-ine) [inline masterlist.yaml]: The version number can be recorded in the commit message, so a comment isn't necessary:

```suggestion
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@cecell, it just requires a few changes to be merged. Also, a note on commit history: while this PR can be squashed & its commit history therefore doesn't need to be spotless, it does help us review your PRs when they're not too messy. There are a few ways you can achieve this.

The first is simply deleting your fork & forking the repo again prior to making your changes & submitting them. Another is to keep your clone of the default branch, `v0.17`, clean by only [merging changes from upstream](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) & creating new branches based on it (e.g., a branch named `cecell-caranthir-tower-reborn`). You could then submit PRs based on those branches, delete them when they're merged, & sync the default branch again. If you wanted to keep using the default branch & keep it clean without deleting your fork, you'd need to use Git commands like `git reset` or `git rebase --interactive` (or equivalents if GitHub Desktop provides them).

Again, you don't need to worry about that for this PR, it's just something to keep in mind for future PRs.

**Comment by cecell** [comment]
Will do, geez I'm trying to get better about the whitespace and the slashes wth.  OK I like the branch idea and will experiment with it later but for now the new fork will be cleaner for everyone.  I have a couple more additions so I'll do that after you commit

**Review by sibir-ine — APPROVED** [review]
@cecell Thanks!

**Pull request merged**